ui/app/styles/components/auth-buttons.scss

```diff
@@ -0,0 +1,22 @@
+.auth-button-tile {
+  height: 31px;
+  width: 31px;
+  background: $white;
+  border-radius: 1px;
+  box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.4);
+}
+.auth-button--google {
+  position: relative;
+  top: -10px;
+  left: -1.05rem;
+}
+
+.auth-button--auth0,
+.auth-button--gitlab {
+  position: relative;
+  top: -6px;
+  left: -0.75rem;
+}
+[class*='auth-button--'] .text {
+  padding-left: $spacing-m;
+}
```

Comment:
Is this BEM looking style used elsewhere in the app?

Reply:
Good catch, mostly no (though I'd like us to) - better to not muddy the waters, I'll change the prefix up.